agenta-backend/agenta_backend/services/container_manager.py

```diff
@@ -127,61 +127,6 @@ def build_image_job(
         raise HTTPException(status_code=500, detail=str(ex))
 
 
-@backoff.on_exception(backoff.expo, (ConnectError, CancelledError), max_tries=5)
```

Comment:
are you sure there is no side effect to this change on the cloud repo?

Reply:
cloud is using a parent method `update_and_sync_templates`